woocommerce/woocommerce-android
# Add jalapeno build flavor

Similar to [this WPAndroid PR](https://github.com/wordpress-mobile/WordPress-Android/pull/11640), this PR adds a new build flavor called “jalapeño”. The eventual goal is to use this flavor for PR builds so testers can install this version along side the current production version and the changes required to update the tooling will come in a later PR since this one is already hefty. The following changes are included in this PR:

- New `jalapeño` build flavor
- Custom launch and woo bubble icons for the `jalapeño` build type
- Splash screens for `wasabi` and `jalapeno`
- New `splash_background` color definition so the jalapeño build could easily override it and for a green splash background. 

### API 23 Screenshots

Icons
--|
<img src="https://user-images.githubusercontent.com/5810477/81907962-c49c7b80-9585-11ea-862d-29fa11fb904f.png" width="300"/>|

Screen | Wasabi | Jalapeño | Release
-- | -- | -- |--
**Splash** |![23-splash-wasabi](https://user-images.githubusercontent.com/5810477/81907996-d8e07880-9585-11ea-8247-2f48a63463f7.png)|![23-splash-jalapeno](https://user-images.githubusercontent.com/5810477/81908004-db42d280-9585-11ea-98db-47e6e57328b4.png)|![23-splash-release](https://user-images.githubusercontent.com/5810477/81908012-de3dc300-9585-11ea-8dcf-1f44ea706ee1.png)
**Login Toolbar** | ![Screenshot_1589443083](https://user-images.githubusercontent.com/5810477/81908381-645a0980-9586-11ea-8c7c-bce222c941ee.png)|![23-login-jalapeno](https://user-images.githubusercontent.com/5810477/81908185-21983180-9586-11ea-9d13-ebf8577e27db.png)|![23-login-release](https://user-images.githubusercontent.com/5810477/81908194-252bb880-9586-11ea-8476-51769c21ae37.png)

### API 28 Screenshots

Icons
--|
<img src="https://user-images.githubusercontent.com/5810477/81909267-99b32700-9587-11ea-9068-dd90c079ef1d.png" width="300"/>|


Screen | Wasabi | Jalapeño | Release
-- | -- | -- |--
**Splash**|![28-splash-wasabi](https://user-images.githubusercontent.com/5810477/81908605-ac792c00-9586-11ea-95bd-cdfeafde80d5.png)|![28-splash-jalapeno](https://user-images.githubusercontent.com/5810477/81908626-b13de000-9586-11ea-901e-ace47e884755.png)|![28-splash-release](https://user-images.githubusercontent.com/5810477/81908633-b3a03a00-9586-11ea-82b4-b28493e6d662.png)
**Login Toolbar**|![28-login-wasabi](https://user-images.githubusercontent.com/5810477/81908766-e77b5f80-9586-11ea-9aa1-a5ca8fccead9.png)|![28-login-jalapeno](https://user-images.githubusercontent.com/5810477/81908778-ec401380-9586-11ea-8558-1e040c9df270.png)|![28-login-release](https://user-images.githubusercontent.com/5810477/81908790-f06c3100-9586-11ea-9c72-fd5de343b67d.png)

## To Test
- update `google-services.json` from `.mobile-secrets`
- Verify the following on release, wasabi and jalapeño builds on a device running API 21 or 22, then again on a device running API 23+:
  - Log in with Google
  - Log in with email address



Update release notes:
